Exiled.Events/Patches/Generic/Scp173BeingLooked.cs

```diff
@@ -36,63 +172,42 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             Label cnt = generator.DefineLabel();
             Label isTutorial = generator.DefineLabel();
 
-            int addCheckOffset = 4;
-            int addCheck = newInstructions.FindLastIndex(instruction => instruction.Calls(Method(typeof(Physics), nameof(Physics.Linecast), new[] { typeof(Vector3), typeof(Vector3), typeof(int) }))) + addCheckOffset;
-
-            LocalBuilder player = generator.DeclareLocal(typeof(Player));
-
+            LocalBuilder currentPlayer = generator.DeclareLocal(typeof(Player));
+            LocalBuilder scp173Player = generator.DeclareLocal(typeof(Player));
```

Comment:
In transpiler, you use only `cnt` label and `scp173Player` local var(but you can do without it, review below), others local vars and labels are useless